modules/AlignmentNewsletterSearch.py

```diff
@@ -147,16 +147,17 @@ def process_message(self, message: ServiceMessage) -> Response:
 
         query = match.group("query")
         return Response(
-            confidence=9, callback=self.process_search_request, args=[query]
+            confidence=9, callback=self.process_search_request, kwargs={"prompt": query}
         )
 
-    async def process_search_request(self, query) -> Response:
+    async def process_search_request(self, prompt: str) -> Response:
         """Search for relevant items for the query.
 
         First we load all items from the Alignment Newsletter database.
         Then we sort the items by relevance to the query.
         Finally we return the most relevant items, if any.
         """
+        query = prompt
```

Comment:
I'll factor out these redundant lines once I know whether I'm on the right track at all.